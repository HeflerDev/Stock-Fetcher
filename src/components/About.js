import React from 'react';

const About = () => (
  <div className="stack">
    <div className="introduction">
      <p>
        This tiny-scale project was developed by
        {' '}
        <a href="https://github.com/heflerdev" target="_blank" rel="noreferrer">
          HeflerDev
        </a>
        .
      </p>
      <p>
        Contact me through
        {' '}
        <a href="https://linkedin.com/in/heflerdev" target="_blank" rel="noreferrer">
          LinkedIn
        </a>
        .
      </p>
      <p>Also, the e-mail heflerdev@gmail.com.</p>
    </div>
  </div>
);

export default About;
