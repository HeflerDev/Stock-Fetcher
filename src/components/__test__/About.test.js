import renderer from 'react-test-renderer';
import About from '../About';

describe('when loading the about page', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<About />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
