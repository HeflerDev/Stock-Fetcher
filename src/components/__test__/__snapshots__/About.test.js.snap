// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when loading the about page renders correctly 1`] = `
<div
  className="stack"
>
  <div
    className="introduction"
  >
    <p>
      This tiny-scale project was developed by
       
      <a
        href="https://github.com/heflerdev"
        rel="noreferrer"
        target="_blank"
      >
        HeflerDev
      </a>
      .
    </p>
    <p>
      Contact me through
       
      <a
        href="https://linkedin.com/in/heflerdev"
        rel="noreferrer"
        target="_blank"
      >
        LinkedIn
      </a>
      .
    </p>
    <p>
      Also, the e-mail heflerdev@gmail.com.
    </p>
  </div>
</div>
`;
